npa/mobilenpa.css

```diff
@@ -0,0 +1,7 @@
+.invisbutton { /*https://stackoverflow.com/questions/13990629/html-css-invisible-button*/
+    background: transparent;
+    border: none !important;
+    /*display: none; - temporarily disabled for tetsing*/
+    pointer-events: none;
+    font-size:50; /*should be 0 for production use*/
+}
```

Comment:
and I would say, add `display: none;` and then add `@media screen and (max-width: 600px) {display: ;}`

Reply:
ok will try